tests/unit/io-writer.spec.js

```diff
@@ -0,0 +1,154 @@
+import _ from 'lodash'
+import fs from 'fs'
+import test from 'tape'
+import Excel from 'exceljs'
+import tempWrite from 'temp-write'
+import Writer from '../../src/io/writer'
+
+const readXlsx = function readXlsx (filePath) {
+  const values = []
+  const workbook = new Excel.Workbook()
+  return workbook.xlsx.readFile(filePath)
+    .then(() => {
+      const worksheet = workbook.getWorksheet(1)
+      let headerLen = null
+      worksheet.eachRow((row) => {
+        let rowValues = row.values
+        rowValues.shift()
+
+        if (headerLen)
+          // older exceljs reader version trims last value
+          // if it is an empty string, so we will pad row
+          // to header length with empty string
+          rowValues = [
+            ...rowValues,
+            ...Array(headerLen - rowValues.length).fill(''),
+          ]
+        else
+          headerLen = rowValues.length
+
+        rowValues = _.map(rowValues,
+          item => (_.isObject(item) || _.isNil(item) ? '' : item))
+        values.push(rowValues)
+      })
+      return values
+    })
+}
+
+/* eslint-disable no-unused-vars */
```

Comment:
Because there is created a `write` instance which is used just to test the configuration validation - for example here:
https://github.com/sphereio/sphere-product-type-export/pull/7/files/146149a15a31ea16a553e7065d087cf5596314bc#diff-578818c88c0b14650abc00dc2e6c8e46R44